src/parser/MarkdownParser.js

```diff
@@ -27,11 +27,10 @@ class MarkdownParser {
         this.params = {};
     }
 
-    parseFile(filePath, options){ // eslint-disable-line no-unused-vars
+    parseFile(filePath) {
```

Comment:
At this point maybe you want to add this :
https://eslint.org/docs/rules/space-before-blocks

To the eslintrc ^^